README.md

```diff
@@ -1,2 +1,71 @@
 # n-ui-foundations
 Sass and js foundations and utilties for the next front-end
+
+## Sass
+```
+@import "n-ui-foundations/main"
+```
+This will output styles for `o-grid`, `o-typography`, `n-util` and `n-ui-normalize`. It will also import mixins for `o-colors`, `o-icons` and `o-visual-effects`. It can be `@imported` multiple times in components or apps with no risk of duplication
+
+### Critical & modular CSS
+Use
+- `nUiStylesheetStart('stylesheet-name')`
+- `nUiStylesheetEnd('stylesheet-name')`
+- `nUiStylesheet('stylesheet-name') {}`
+These insert comments that, when built with `n-ui`'s build tools, will split your stylesheet into multiple sub stylesheets. Can be used for critical css (e.g. use `head-main` as the stylesheet name for the same effect as `n-ui@3`'s `nUiCritical()` mixin)
```

Comment:
yep. n-ui builds head-main-n-ui-core (or something like that), but apart from that it's only stream-page that has more variants, and I spoke to Andy about stylesheets before implementing any of this, so hopefully no surprises there